### DISCUSSION TITLE
config files not working

### DISCUSSION THREAD

**Discussion by superjump071** [itzg/docker-minecraft-server#1827] [Q&A] [answered]
i am trying to run  [mustard virus by forgelabs](https://www.curseforge.com/minecraft/modpacks/zombie-apocalypse-slow-zombies-by-forge-labs).
the newest version does not have a server files modpack so im trying to make it work with the client mod pack and the mods work but the config does not. here is my .yml file i cant find a problem
`version: "3"

services:
  minecraft:
    image: itzg/minecraft-server:java8-multiarch
    volumes:
      - minecraftdata:/data
      - E:/server files/mods:/mods
      - E:/server files/config:/config
      ports:
      - "25565:25565"
    environment:
      WORLD:
      EULA: "TRUE"
      VERSION: "1.12.2"
      TYPE: "FORGE"
      FORGE_INSTALLER: "forge-1.12.2-14.23.5.2855-installer.jar"
      OPS: "Airrunner"
      SERVER_PORT: "25565"
      DEBUG_EXEC: "True" 
      MEMORY: "4G"
    tty: true
    stdin_open: true
    restart: unless-stopped
volumes:
  minecraftdata:
    external: true`

what am i doing wrong 
for reference here are some screenshots of my file system
![afbeelding](https://user-images.githubusercontent.com/88433894/201930545-1ecfe129-5129-445a-96c1-2689c22923fc.png)
![afbeelding](https://user-images.githubusercontent.com/88433894/201930616-620a14c3-42f9-4ab6-8349-4b04decc100f.png)
![afbeelding](https://user-images.githubusercontent.com/88433894/201930666-27a92862-13ff-4926-a96f-671b2470e4a4.png)

**Comment by itzg** [comment, marked as answer]
If they're config files for the plugins specifically then you'll need to put them also in the mounted plugins directory. Here is an example where I discovered this recently:

https://github.com/itzg/docker-minecraft-server/tree/master/examples/multi-project/servers
